### ISSUE TITLE
Jenkins pipeline syntax errors not caught

### ISSUE THREAD

**Issue by PJHRobles** [nvuillam/npm-groovy-lint#52]
It doesn't look like syntax errors are caught. If I have the following code below:
```
pipeline {
  agent any
```
and changed it to something which is an error:
```
pipeline_asdasd_dasdasdasd_asdasdas {
  agent any
```
It doesn't seem to catch it as an error (it doesn't catch it at all) when it is.

**Comment by nvuillam** [comment]
Your second exemple remains valid groovy code, that's why no error is found
There is a Jenkinsfile linter included in jenkins servers (not sure a similar tool exists locally), you can see its IDE integration there: 

https://www.jenkins.io/doc/book/pipeline/development/#linter

There is an existing CLI npmbpackage : https://www.npmjs.com/package/jflint , but you still need a reachable jenkins server

Maybe someday i could embed such package in npm-groovy-lint so you could have a "all-in-one" tool... :)

**Comment by PJHRobles** [comment]
Ack. Thanks for explaining!
